oap-server/server-starter/src/main/resources/meter-analyzer-config/satellite.yaml

```diff
@@ -13,22 +13,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-expSuffix: tag({tags -> tags.service = 'satellite::' + tags.service}).instance(['service'], ['instance'])
+expSuffix: tag({tags -> tags.service = 'satellite::' + tags.service}).service(['service'])
```

Comment:
I think `instance` downsampling function would generate both service and service instance metrics, according to MAL doc.

Reply:
I think it is better to just rename the metric(removing `instance_`). Or do you consider instance level metrics are not useful later? I was planning to consider Satellite so11y should be a part of the SkyWalking so11y in the v9 new dashboard.

Reply:
There is a one-to-one correspondence between the metric names with the entity level, and I see the MAL could only generate the same name metric, why it could generate the service level metric? Do I miss anything?

Reply:
Metric name has nothing related to the generated result.

I said so according to the doc, https://skywalking.apache.org/docs/main/v8.9.1/en/concepts-and-designs/mal/#metric-level-function

Reply:
But when I query the metrics without the instance argument, the data is always zero.
Such as:
```shell
# could generate the result
swctl metrics linear --name satellite_instance_server_cpu_utilization --service-name satellite::satellite-service --instance-name satellite-instance
# no data responded
swctl metrics linear --name satellite_instance_server_cpu_utilization --service-name satellite::satellite-service
```

Reply:
@wankai123 Should I consider this is a bug? `SampleFamily#buildMeterEntity` doesn't return a List, such as in instance case. But the document says otherwise.

Reply:
OK, I am following up the context. This PR is good, the MAL doc should be updated.